crafting_combinator/control.lua

```diff
@@ -51,6 +51,7 @@ script.on_load(on_load)
 script.on_configuration_changed(function(changes)
 	update_bottleneck(changes)
 	enable_recipes()
+	rc_control.init_global()
```

Comment:
This really shouldn't be called here, it's intended only for the initial setup. If there's any setup to do on version changes, put that into a one-time migration instead. Though with my other suggestion, this shouldn't be necessary at all.